<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tela_criar_conta</class>
 <widget class="QMainWindow" name="tela_criar_conta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>263</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Criar Conta</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icone_login.png</normaloff>icone_login.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="botao_criar_conta">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string notr="true">Criar conta</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="texto_usuario">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>19</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="maxLength">
     <number>40</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_nome">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string notr="true">Nome:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="texto_senha_1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>79</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <strikeout>false</strikeout>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_senha_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string notr="true">Senha:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_senha_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>111</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string notr="true">Confirmar senha:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="texto_senha_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <strikeout>false</strikeout>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_email">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>51</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string notr="true">Email:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="texto_email">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="maxLength">
     <number>40</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="group_box_cargo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>401</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string notr="true">Cargo</string>
    </property>
    <widget class="QRadioButton" name="radio_button_vendedor">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">Vendedor</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_button_gerente">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">Gerente</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_button_entregador">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">Entregador</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
